src/main/scala/strawman/collection/mutable/ArrayDeque.scala

```diff
@@ -0,0 +1,349 @@
+package scala                     //TODO: Change this strawman --> we just need sizeHintIfCheap from scala
```

Comment:
Can we actually do this?  I don't think we're likely to add any new collections before strawman hits.